rhobs/alerting/data_plane/prometheus.high_memory_usage_alert.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-pod-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: pod_alert_high_memory_usage
+    interval: 5m
+    rules:
+    - alert: container_cpu_usage_seconds_total
+      expr: sum by (pod, namespace)(rate(container_cpu_usage_seconds_total{container="",pod!=""}[5m])){}
+      for: 5m
+      labels:
+        severity: warning
+      annotations:
+        summary: pod {{ $labels.pod}} pod identified with memory usage above 90%
```

Comment:
the rate above is for cpu usage seconds...this summary seems wrong.

Reply:
Also, I guess I'm a little confused, but should the thresholds be given in the `expr` lines?

Reply:
Example: https://github.com/redhat-appstudio/o11y/blob/main/rhobs/alerting/data_plane/prometheus.release_service_alerts.yaml#L30